docs/api/app.md

```diff
@@ -580,6 +580,10 @@ To set the locale, you'll want to use a command line switch at app startup, whic
 
 **Note:** On Windows you have to call it after the `ready` events gets emitted.
 
+### `app.getRegion()` _macOS_ _Linux_ _Windows_
```

Comment:
For functions that work across platforms, no need to specify all of `_macOS_ _Linux_ _Windows_`